README.md

```diff
@@ -7,9 +7,33 @@ This includes:
 * Intel SGX SDK (using Docker to run the installer)
 * Rust SGX SDK headers and EDL files (using a sparse Git checkout)
 
+## Prerequisites
+
+1. Install [Docker](https://docs.docker.com/get-docker/)
+2. Docker [Post-Installation](https://docs.docker.com/engine/install/linux-postinstall/) Steps
+    * Create the docker group.
+
+    ```
+    sudo groupadd docker
+    ```
+
+    * Add your user to the docker group.
+
+    ```
+    sudo usermod -aG docker $USER
+    ```
+
+3. [Enable](https://docs.docker.com/develop/develop-images/build_enhancements/#to-enable-buildkit-builds) Docker Buildkit builds
```

Comment:
Good catch!

It's actually a good idea to list the right version of Git here too. I might also suggest just linking to upstream Docker documentation, rather than elaborating on platform-specific Docker instructions here::

```suggestion
1. Git 2.25+ (for sparse checkout support)
2. [Docker](https://docs.docker.com/get-docker/)
```

For BuildKit, it's probably better to just have the script itself explicitly enable it, rather than depending on the host, and having to document that here; I merged a quick change for that:

- #3